app/controllers/admin/items_controller.rb

```diff
@@ -80,4 +86,10 @@ def destroy_flash_and_redirect(item)
     redirect_to "/admin/merchants/#{params[:merchant_id]}/items"
   end
 
+  def fulfill_update_inventory(item, order)
+    item.fulfill_item(order)
+    item.update_inventory(order)
+    flash[:notice] = "You have fulfilled the order for #{item.name}"
+    redirect_to "/admin/merchants/#{item.merchant.id}/orders/#{order.id}"
```

Comment:
Add method for refactor purposes that calls two other item model methods, shows a flash message and redirects